src/scripts/utils.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+window.convertNumberToSalary = (num) => {
+  return `$${(+num).toLocaleString('en-US')}`;
+};
+
+window.convertSalartyToNumber = (a) => {
```

Comment:
There is a typo in your function name `convertSalartyToNumber`. It should be `convertSalaryToNumber`. Always try to keep your function names meaningful and free from typos, it helps in maintaining the readability and understanding of your code.